lang/java/reef-applications/reef-vortex/src/main/java/org/apache/reef/vortex/driver/VortexDriver.java

```diff
@@ -76,7 +79,9 @@ private VortexDriver(final EvaluatorRequestor evaluatorRequestor,
                        @Parameter(VortexMasterConf.WorkerMem.class) final int workerMem,
                        @Parameter(VortexMasterConf.WorkerNum.class) final int workerNum,
                        @Parameter(VortexMasterConf.WorkerCores.class) final int workerCores,
-                       @Parameter(VortexMasterConf.NumberOfVortexStartThreads.class) final int numOfStartThreads) {
+                       @Parameter(VortexMasterConf.NumberOfVortexStartThreads.class) final int numOfStartThreads,
+                       @Parameter(VortexMasterConf.InputCodec.class) final Codec inputCodec,
```

Comment:
Thanks a lot for the comment! I'll update the PR and reopen it for another review.